### ISSUE TITLE
Able to use crazyradio with VM, but not standalone cfclient 

### ISSUE THREAD

**Issue by spohorec** [bitcraze/crazyflie-clients-python#253]
I'm running Ubuntu 14.04 and trying to control a Crazyflie2.0 from a Crazyradio PA. I've gotten it to work on the Bitcraze VM, but no matter what can't seem to make it work outside of that. 

I updated the firmware from inside the VM, and when I add the device it appears as "Bitcraze Crazyradio (PA) USB Dongle." Outside of the VM, it's lsusb entry is `Bus 001 Device 010: ID 1915:7777 Nordic Semiconductor ASA`

I also attempted to update the firmware outside of the VM, which told me it was successful, but didn't appear to make any difference to cfclient. It also changed the lsusb entry to `Bus 001 Device 010: ID 1915:0101 Nordic Semiconductor ASA` if that matters.

When I run cfclient, I get the following:
```
spohorec@Sam:~$ cfclient
INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Found readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Successfully initialized [linuxjsdev]
INFO:cfclient.utils.input.inputreaders:Could not initialize [pysdl2]: No SDL2 support on Linux
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
INFO:cfclient.utils.config:Dist config read from /usr/local/lib/python3.4/dist-packages/cfclient/configs/config.json
INFO:cfclient.utils.config:Config file read from [/home/spohorec/.config/cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:cfclient.utils.zmq_param:Biding ZMQ for parameters at tcp://*:1213
INFO:cfclient.utils.zmq_led_driver:Biding ZMQ for LED driverat tcp://*:1214
INFO:cfclient.utils.input:Using device blacklist [(VirtualBox|VMware)]
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cfclient.utils.input:Selected MUX: Normal
INFO:cfclient.ui.main:Role of Gasia Co.,Ltd PS(R) Gamepad is Device
INFO:cfclient.utils.input.mux:Adding device Gasia Co.,Ltd PS(R) Gamepad to MUX Normal
``` 
If I hit "scan" in the GUI, I get `INFO:cflib.drivers.cfusb:Looking for devices....`

I'm kind of stumped on what to try next at this point, so any help would be appreciated! Thanks!

**Comment by ataffanel** [comment]
Hi,
have you tried to run the client with sudo? This could be an access right problem. If so you can fix it by adding udev rules: https://github.com/bitcraze/crazyflie-lib-python#linux

**Comment by spohorec** [comment]
I have tried with sudo, and received the same results. I've already tried updating the udev, I have two entries:
`SUBSYSTEM=="usb", ATTRS{idVendor}=="1915", ATTRS{idProduct}=="7777", MODE="0664", GROUP="plugdev"` (suggested by the tutorial you linked)
and
`SUBSYTEM=="usb",ATTR(idVendor}=="1915",GROUP="dialout"` (suggested by someone else) 

It still won't pick up the radio.

**Comment by cstanke** [comment]
Do you have Crazyradio or Crazyradio PA? They have different product IDs. Try "0101" instead...?

**Comment by ataffanel** [comment]
They are not supposed to have different PIDs. @cstanke: have you seen that on any of your crazyradios?

@spohorec: What does the client about box says? We have a diagnostic text field in there.

**Comment by cstanke** [comment]
@arnaud: I'm sorry, I made an assumption. There are two different PIDs listed [here](https://wiki.bitcraze.io/projects:virtualmachine:create_vm?s[]=udev#adding_udev_rules_for_crazyradio) and I assumed one was for the PA model.

I should be more sure of my answers before interjecting!

My appologies.

**Comment by ataffanel** [comment]
@cstanke No worries :). I got a bit concerned because if there is any badly programmed CR out there I want to know about it. 0x0101 is actually for the Nordic USB bootloader, to flash the Crazyradio firmware.

**Comment by spohorec** [comment]
I do in fact have a PA. And actually, adding a product ID of 0101 in a different udev rule got my radio to work.  
@ataffanel I've noticed 0101 show up as the product ID sometimes. Usually, it showed up when I ran the bootloader from the command line (outside of VM) and flashing firmware. I believe it goes away and returns as 7777 when I remove and reinsert the radio.

It's working now, though we'll see if I can repeat the results on the next computer I need to install cfclient on (will be doing that this afternoon). It's possible changing the product ID made no difference, but I did something else at the same time that was the solution. If it does something interesting I'll update the thread, but my issue seems to be resolved!
